### DISCUSSION TITLE
Any chance to get a Serbian Bible translation in Obsidian?

### DISCUSSION THREAD

**Discussion by ademiter01** [tim-hub/obsidian-bible-reference#222] [Q&A]
Hi Team,
I am so grateful for this plugin, it's amazing for notes with Bible verses.

My question is - how hard is it to add a new translation into the plugin and what are the pre-reuqisites? I work in an IT role myself and would love to assist if I can in any way. I'd be glad to recommend the app to all my Christian friends and leaders in Serbia.

Thanks!

**Comment by tim-hub** [comment]
Translations are mainly based on 2 API providers, you can find them here.

https://github.com/tim-hub/obsidian-bible-reference/blob/master/docs/bible-api-and-source.md

**Comment by AndyPandy1189** [comment]
I actually reached out to the Bolls Life API provider and he got the NABRE translation added for me which then flowed downstream, so you could try that avenue to get it added to the plugin. He was very responsive and helpful!
